### DISCUSSION TITLE
Welcome to New Relic Terraform Discussions!

### DISCUSSION THREAD

**Discussion by kidk** [newrelic/terraform-provider-newrelic#1478] [Announcements]
## 👋 Welcome!
The New Relic Terraform community will Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas
  * Talk about roadmap and new resources
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rdhar** [comment]
Hey, thanks for enabling discussions. Hoping to have broader channels of communication beyond those related to issues: particularly when there are newly released features.

For background, I'm the point-of-contact for managing/instrumenting New Relic monitoring of our environments worldwide. Day-to-day, it involves: setting up NRQL alerts, composing dashboards and investigating anomalies with APM and Synthetics. Since we operate at an enterprise-level, I've started leaning on Terraform for large-scale orchestration of alerts, dashboards and workloads across thousands of apps within dozens of (sub-)accounts.

Recent developments in NRQL alerts and consolidating dashboard resources has been great standardisation of our setup. Looking forward to more improvements along the way.
